Snowflake.Data.Tests/UnitTests/Logger/EasyLoggerManagerTest.cs

```diff
@@ -92,6 +93,38 @@ public void TestThatLogsToProperFileWithProperLogLevelOnly()
             Assert.That(logLines, Has.Exactly(1).Matches<string>(s => s.Contains(FatalMessage)));
         }
         
+        [Test]
+        public void TestThatOnlyUnknownFieldsAreLogged()
+        {
+            // arrange
+            string expectedFakeLogField = "fake_log_field";
+            string ConfigWithUnknownFields = $@"{{
+                    ""common"": {{
+                        ""LOG_LEVEL"": ""warn"",
+                        ""lOg_PaTh"": ""path"",
+                        ""{expectedFakeLogField}_1"": ""abc"",
+                        ""{expectedFakeLogField}_2"": ""123""
+                    }}
+                }}";
+            var configFilePath = Guid.NewGuid().ToString() + ".json";
+            using (var writer = File.CreateText(configFilePath))
+            {
+                writer.Write(ConfigWithUnknownFields.Replace("\\", "\\\\"));
+            }
+            EasyLoggerManager.Instance.ReconfigureEasyLogging(EasyLoggingLogLevel.Warn, t_directoryLogPath);
+
+            // act
+            var parser = new EasyLoggingConfigParser();
```

Comment:
I would move creating a parser to `// arrange` section because it is a preparation for the main action

Reply:
Ok

Reply:
Fixed